From f39cf5251953b468e9d921e1cf2aca3abdb00772 Mon Sep 17 00:00:00 2001
From: Damien Regad <dregad@mantisbt.org>
Date: Wed, 24 Jun 2015 14:52:36 +0200
Subject: [PATCH] Change default threshold to view project doc to VIEWER

Previously it was ANYBODY, which would let any user download files from
any project including private ones, even when they are not part of the
team.

Backport from a4be76d6e5c4939545d84712c79d3f8f4a108c4f

Fixes #19873
---
 config_defaults_inc.php | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/config_defaults_inc.php b/config_defaults_inc.php
index 0ef20fece7..374ab13240 100644
--- a/config_defaults_inc.php
+++ b/config_defaults_inc.php
@@ -2354,9 +2354,13 @@
 
 	/**
 	 * Threshold needed to view project documentation
+	 * Note: setting this to ANYBODY will let any user download attachments
+	 * from private projects, regardless of their being a member of it.
+	 * @see $g_enable_project_documentation
+	 * @see $g_upload_project_file_threshold
 	 * @global int $g_view_proj_doc_threshold
 	 */
-	$g_view_proj_doc_threshold = ANYBODY;
+	$g_view_proj_doc_threshold = VIEWER;
 
 	/**
 	 * Site manager
